ParabolInc/parabol
# Add generic createTaskIntegration mutation

Resolves #5819

**How to test:**
- Go to the team board
- Add jira integration
- Create a task on the team board
- Push a task to Jira
- See it pushed successfully
- Push someone else (another user) task to Jira, see a comment added on Jira
- See it pushed successfully
- Do the same for GitHub
- Check that pushing tasks to integrations works in the demo
